File: src/main/java/uk/co/threebugs/mochiwhattotrade3/setup/SetupDto.java
package uk.co.threebugs.mochiwhattotrade3.setup;

import lombok.Builder;
import lombok.Value;
import lombok.extern.jackson.Jacksonized;
import uk.co.threebugs.mochiwhattotrade3.setupgroup.SetupGroupDto;

@Builder
@Value
@Jacksonized
public class SetupDto {
    Integer id;
    SetupGroupDto setupGroup;
    String symbol;
    Integer rank;
    Integer dayOfWeek;
    Integer hourOfDay;
    Integer stop;
    Integer limit;
    Integer tickOffset;
    Integer tradeDuration;
    Integer outOfTime;
}

File: src/main/kotlin/uk/co/threebugs/mochiwhattotrade3/setup/SetupDto.kt
package uk.co.threebugs.mochiwhattotrade3.setup

import uk.co.threebugs.mochiwhattotrade3.setupgroup.SetupGroupDto

data class SetupDto (
    var id: Int? = null,
    var setupGroup: SetupGroupDto? = null,
    var symbol: String? = null,
    var rank: Int? = null,
    var dayOfWeek: Int? = null,
    var hourOfDay: Int? = null,
    var stop: Int? = null,
    var limit: Int? = null,
    var tickOffset: Int? = null,
    var tradeDuration: Int? = null,
    var outOfTime: Int? = null
)
